ps_mbo.php

```diff
@@ -135,6 +135,42 @@ protected function installTabs()
 
         return true;
     }
+    
+    public function fetchModules($controller) {
+        $controllerWhiteList = array('AdminCarriers', 'AdminPayment');
+        if (!in_array($controller, $controllerWhiteList)) {
+            return false;
+        }
+        
+        $panel_id = '';
+        $modules = array();
+        $data = array();
+        
+        switch ($controller) {
+            case 'AdminCarriers':
+                $modules = $this->getCarriersMboModules();
+                $data['panel_id'] = 'recommended-carriers-panel';
+                $data['panel_title'] = $this->trans('Use one of our recommended carrier modules', [], 'Admin.Shipping.Feature');
+                break;
+            case 'AdminPayment':
+                $modules = $this->getPaymentMboModules();
+                break;
+            default: // should never happen
+                exit;
```

Comment:
It seems to be unused. Thanks to your next `if`, you will return if because the variable is empty.